bin/sync_cms_to_repo.sh

```diff
@@ -9,14 +9,17 @@ npx tsx bin/sync_cms_to_plugins_repo.js
 
 echo "Creating PR \"$PR_TITLE\" for branch $BRANCH_NAME"
 
-// This is the only file we want to commit
+# This is the only file we want to commit
```

Comment:
I had blinders on and couldn't figure out why I was getting errors about a directory. TLDR; JS comments in Bash. 🙃